README.md

```diff
@@ -2,9 +2,9 @@
 
 Tools for Azure Pipelines. These tools allow a Toolkit developer to validate their code using Azure Pipelines, ensuring the code is formatted properly, does not contain ambiguous statements and passes all the tests on every single commit.
 
-# How to use these tools
+# How to use the build system.
 
-Create a file named  `azure-pipeline.yml` file at the root of your repo and put the following:
+Create a file named `azure-pipelines.yml` file at the root of your repo and put the following in it:
```

Comment:
Should we also mention creating / copying `.pre-commit-config.yaml` in the README?  Now that code-style-validation relies on the pre-commit hook, this seems like something worth explaining in the documentation.

Reply:
Done